src/main.js

```diff
@@ -0,0 +1,31 @@
+import { getProductList } from './ProductService.js';
+import { Product } from './Product.js';
+import { ElectronicProduct } from './ElectronicProduct.js';
+
+async function main() {
+  try {
+    const data = await getProductList(1, 10, "");
+
+    if (!data || !data.items || data.items.length === 0) {
```

Comment:
API 문서에 보면 data.items으로 되어있지 않고 data.list로 되어있더라구요
이 부분을 data.list로 수정해주시면 올바르게 동작할 것 같네요~

```suggestion
    if (!data || !data.list || data.list.length === 0) {
```